proto/raft_cmdpb.proto

```diff
@@ -246,6 +246,8 @@ message RaftCmdRequest {
     repeated Request requests             = 2; 
     optional AdminRequest admin_request   = 3; 
     optional StatusRequest status_request = 4;
+
+    optional bool sync_log                = 5;
```

Comment:
I prefer to move this to the request header.

Reply:
should we sync_log for admin_request?  eg. conf_change, split

Reply:
I think we should do. 
Maybe we should also sync compact log, but I worry this may hurt the performance.

Reply:
/cc @BusyJay

Reply:
I don't see any reason to sync compact log.

Reply:
we can distinguish admin_request type in tikv,  so it's no need to let pd known sync_log,  will move sync_log to request header.